Formula/micronaut.rb

```diff
@@ -0,0 +1,21 @@
+class Micronaut < Formula
+  desc "Modern JVM-based framework for building modular microservices"
+  homepage "http://micronaut.io"
+  url "https://github.com/micronaut-projects/micronaut-core/releases/download/v1.0.0.M3/micronaut-1.0.0.M3.zip"
+  version "1.0.0.M3"
+  sha256 "394a315d0eeeb567b41a3ddfb9cad374e7b934ac9ba69ffb53fa38d0ea355e79"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    rm_f Dir["bin/*.bat"]
+    libexec.install %w[bin media cli-1.0.0.M3.jar]
+    bin.install_symlink libexec/"bin/mn"
+  end
+
+  test do
+    assert_match "Micronaut CLI command line interface for generating projects and services", shell_output("#{bin}/mn --help")
```

Comment:
This should be wrapped to 80 characters.

Reply:
Also when ILZ does see this you're likely to be asked for some kind of stronger functionality testing rather than a simple test like this.

Reply:
Why bother add more testing here if the codebase already performs the necessary unit/integration tests? 

AFAIK making sure the executable works should be enough.

Reply:
Because ultimately if Homebrew ships something broken to users it's Homebrew that gets dumped on.

Proper testing in `brew`, ran through multiple CI environments, is an appreciably saner system for users & maintainers than shrugging and assuming upstream will always diligently test against the latest Java and the three latest major macOS releases.

Reply:
@gastaldi Maybe just running `mn create-app hello-world` and asserting the directory is created. That represents more of a functional tests, which seems to be what @DomT4 is after

Reply:
@graemerocher That sounds reasonable enough to me at least.